src/hooks/useSecureContext.ts

```diff
@@ -0,0 +1,6 @@
+const useSecureContext = () => {
+  // TODO: (Punksage): After testing return it to window.isSecureContext
```

Comment:
Is it for testing?

Reply:
Exactly. This is for the testing. 

If you wish you can trigger the Copy/Download button functionality without changing http-https protocol to server Testkube. I've left it for you so other developers can discover the feature or verify if it is working correctly.